components/inputs/demo/input-radio.html

```diff
@@ -6,47 +6,89 @@
 		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
 		<script type="module">
 			import '../../demo/demo-page.js';
-			import '../input-radio-spacer.js';
+			import '../../form/form.js';
+			import '../input-group.js';
+			import '../input-radio.js';
+			import '../input-radio-group.js';
+			import '../../inputs/input-text.js';
 			import './input-radio-label-test.js';
-			import './input-radio-label-simple-test.js';
-			import './input-radio-solo-test.js';
-			import './input-radio-spacer-test.js';
```

Comment:
I'm de-emphasizing the old native radio demos, but kept one of them and the vdiffs.